test/e2e/scheduling/limit_range.go

```diff
@@ -222,6 +226,89 @@ var _ = SIGDescribe("LimitRange", func() {
 		framework.ExpectNoError(err)
 	})
 
+	ginkgo.It("should ensure that a limitRange can be listed, patched and deleted by collection", func() {
```

Comment:
`It` refers to the feature (LimitRange), not the test. So it should be something like:

```golang
ginkgo.It("should list, patch and delete a LimitRange by collection")
```